static/app/views/settings/project/dynamicSampling/dynamicSamping.spec.tsx

```diff
@@ -422,7 +422,7 @@ describe('Dynamic Sampling', function () {
           dynamicSampling: {
             rules: [
               TestStubs.DynamicSamplingConfig().uniformRule,
-              TestStubs.DynamicSamplingConfig().specificRule,
+              {...TestStubs.DynamicSamplingConfig().specificRule},
```

Comment:
I think this was there because I needed to update the sampleRate to be lower than the uniform rule sample rate, but in the end, I decided to update the `testsStubs` and forgot to update here ...thanks